Source/Examples/WPF.SharpDX/DeferredShadingDemo/DeferredShadingDemo_NET40.csproj

```diff
@@ -45,9 +46,11 @@
       <HintPath>..\..\..\packages\PropertyTools.Wpf.2014.1.29.1\lib\NET40\PropertyTools.Wpf.dll</HintPath>
     </Reference>
     <Reference Include="SharpDX">
-      <HintPath>..\..\..\packages\SharpDX.2.5.0\lib\net40\SharpDX.dll</HintPath>
+      <HintPath>$(SharpDXPackageBinDir)\SharpDX.dll</HintPath>

```

Comment:
This is an environment variable set by SharpDX's nuget package install script. It adds a `.targets` reference to the project to resolve that path. It works locally, I'm still trying to figure out why it doesn't work on AppVeyor.